### ISSUE TITLE
Unity bonus/benefit of squares, and criteria for usage, is undocumented.

### ISSUE THREAD

**Issue by Swift016** [MaFi-Games/Captain-of-Industry-issues#1255]
### Issue description

I've played for 370 hours and only just learned that settlement squares grant a unity bonus via the wiki.

This is problematic from a UX standpoint, as:

- Squares do not have a description outlining their purpose in the tooltip (their tooltip is blank).
- The settlement popup (when clicking on housing buildings) does not indicate the missing "settlement quality" criteria (it really should show 0/1.00 if the need is going unmet).
- The unity popup/population overview does not indicate "settlement quality" at all unless a square is built.
- The game does not explain the criteria for meeting 100% of the quality need (that every house needs to be attached to a square). It would also be sensible to explain that both adjacent and diagonal counts toward the criteria.
- The message center article on "Population and Unity" does not go into enough detail on unity in general, including re: squares.

### Reproduction steps

N/A

### Game version and build number

v0.5.4g (b244)

### Operating system

Windows

**Comment by JamieRCHI** [comment]
Yes. I always wondered what they are good for. I never use them. Add info to Wiki? hint hint
